From 294c52f4a61df2571fad5f69f7f2257270ce24ae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Wed, 2 Nov 2022 13:35:50 +0000
Subject: [PATCH 04/11] [openmp] [test] Use omp_testsuite.h instead of directly
 including pthread.h

OpenMP tests that use pthread functions include this header instead.
On Unix systems, this header includes pthread.h, while it provides
minimal implementations of the used pthread functions for Windows.

Differential Revision: https://reviews.llvm.org/D137746
---
 openmp/runtime/test/tasking/bug_nested_proxy_task.c      | 2 +-
 openmp/runtime/test/tasking/bug_proxy_task_dep_waiting.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/openmp/runtime/test/tasking/bug_nested_proxy_task.c b/openmp/runtime/test/tasking/bug_nested_proxy_task.c
index b4e8a5460287..44e754bef044 100644
--- a/openmp/runtime/test/tasking/bug_nested_proxy_task.c
+++ b/openmp/runtime/test/tasking/bug_nested_proxy_task.c
@@ -8,7 +8,7 @@
 
 #include <stdio.h>
 #include <omp.h>
-#include <pthread.h>
+#include "omp_testsuite.h"
 #include "omp_my_sleep.h"
 
 /*
diff --git a/openmp/runtime/test/tasking/bug_proxy_task_dep_waiting.c b/openmp/runtime/test/tasking/bug_proxy_task_dep_waiting.c
index d48314ba1408..7737a0ef7c67 100644
--- a/openmp/runtime/test/tasking/bug_proxy_task_dep_waiting.c
+++ b/openmp/runtime/test/tasking/bug_proxy_task_dep_waiting.c
@@ -8,7 +8,7 @@
 
 #include <stdio.h>
 #include <omp.h>
-#include <pthread.h>
+#include "omp_testsuite.h"
 #include "omp_my_sleep.h"
 
 /*
-- 
2.25.1

